examples/i2c/client_pn532.c

```diff
@@ -30,15 +38,23 @@ cothread_t t_main;
 #define STACK_SIZE (4096)
 static char t_client_main_stack[STACK_SIZE];
 
-bool read_passive_target_id(uint8_t card_baud_rate, uint8_t *uid_buf, uint8_t *uid_buf_length, uint8_t timeout) {
+bool read_passive_target_id(uint8_t card_baud_rate, uint8_t *uid_buf, uint8_t *uid_buf_length, uint8_t timeout)
+{
     uint8_t cmd_header[3] = { PN532_CMD_INLISTPASSIVETARGET, 1, card_baud_rate };
 
-    int ret = pn532_write_command(cmd_header, 3, NULL, 0, DEFAULT_READ_ACK_FRAME_RETRIES);
-    if (ret < 0) return false;
+    uint8_t write_fail = pn532_write_command(cmd_header, 3, NULL, 0, DEFAULT_READ_ACK_FRAME_RETRIES);
+    if (write_fail) {
+        LOG_CLIENT_ERR("failed to write PN532_CMD_INLISTPASSIVETARGET!\n");
+        return false;
+    };
 
     uint8_t response[64];
-    bool response_ret = pn532_read_response(response, 64, DEFAULT_READ_RESPONSE_RETRIES);
-    if (!response_ret) return false;
+    uint8_t read_fail = pn532_read_response(response, 64, DEFAULT_READ_RESPONSE_RETRIES);
+    if (read_fail) {
+        // LOG_CLIENT_ERR("failed to read response for PN532_CMD_INLISTPASSIVETARGET!\n");
```

Comment:
Should this be commented out?

Reply:
It just pollutes output because its what fails when card Reader times out which is very frequent